packages/calcite-components/src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -611,255 +412,370 @@ describe("calcite-input-time-picker", () => {
-      await page.keyboard.press("Backspace");
-      await page.keyboard.type("16.30.13");
-      await page.keyboard.press("Tab");
-      await page.waitForChanges();
+        const inputTimePicker = await page.find("calcite-input-time-picker");
+        const input = await page.find("calcite-input-time-picker >>> calcite-input-text");
 
-      expect(changeEvent).toHaveReceivedEventTimes(2);
-      expect(await inputTimePicker.getProperty("value")).toBe("16:30:13");
-      expect(await getInputValue(page)).toBe("16.30.13");
-    });
-  });
+        const date = new Date(0);
+        date.setHours(13);
+        date.setMinutes(59);
+        date.setSeconds(59);
 
-  describe("australian english locale support", () => {
-    it("allows typing in australian english format", async () => {
-      const page = await newE2EPage();
-      await page.setContent(`<calcite-input-time-picker step="1" lang="en-AU"></calcite-input-time-picker>`);
+        const expectedValue = date.toISOString().substr(11, 8);
+        const expectedInputValue = localizeTimeString({ value: expectedValue, locale, numberingSystem });
 
-      const inputTimePicker = await page.find("calcite-input-time-picker");
-      const changeEvent = await inputTimePicker.spyOnEvent("calciteInputTimePickerChange");
+        inputTimePicker.setProperty("value", expectedValue);
 
-      await inputTimePicker.callMethod("setFocus");
-      await page.waitForChanges();
-      await page.keyboard.type("2:3:5 am");
-      await page.keyboard.press("Enter");
-      await page.waitForChanges();
+        await page.waitForChanges();
 
-      expect(await getInputValue(page)).toBe("02:03:05 am");
-      expect(changeEvent).toHaveReceivedEventTimes(1);
-      expect(await inputTimePicker.getProperty("value")).toBe("02:03:05");
+        const inputValue = await input.getProperty("value");
+        const inputTimePickerValue = await inputTimePicker.getProperty("value");
 
-      await page.keyboard.press("Backspace");
-      await page.keyboard.press("Backspace");
-      await page.keyboard.press("Backspace");
-      await page.keyboard.press("Backspace");
-      await page.keyboard.press("Backspace");
-      await page.keyboard.press("Backspace");
-      await page.keyboard.press("Backspace");
-      await page.keyboard.press("Backspace");
-      await page.keyboard.press("Backspace");
-      await page.keyboard.press("Backspace");
-      await page.keyboard.press("Backspace");
-      await page.keyboard.type("4:3:5 pm");
-      await page.keyboard.press("Tab");
-      await page.waitForChanges();
+        expect(inputValue).toBe(expectedInputValue);
+        expect(inputTimePickerValue).toBe(expectedValue);
+      });
+    });
 
-      expect(await getInputValue(page)).toBe("04:03:05 pm");
-      expect(await inputTimePicker.getProperty("value")).toBe("16:03:05");
-      expect(changeEvent).toHaveReceivedEventTimes(2);
+    supportedLocales.forEach((locale) => {
```

Comment:
One more thing, these tests supersede the [locale-specific render ones above](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/components/input-time-picker/input-time-picker.e2e.ts#L36-L51). We can remove those now in favor of these new ones.